apps/docusaurus/docs/move/move-on-aptos/scripts/writing-scripts.md

```diff
@@ -0,0 +1,58 @@
+---
+title: "Writing Move Scripts"
+---
+
+import CodeBlock from '@theme/CodeBlock';
+
+# How can I write Move Scripts?
+
+Move scripts can be written in tandem with Move contracts, but it's highly
+suggested to use a separate Move package for it.
+
+## Package layout
+
+The package needs a Move.toml and a sources directory, similar to code modules.
+
+For example, we may have a directory layout like:
+
+```
+my_project/
+|-> Move.toml
+ \  sources/
+  |-> my_script.move
```

Comment:
can you write the layout in [this style](https://doc.rust-lang.org/cargo/guide/project-layout.html)?